Gui/opensim/view/src/org/opensim/view/Installer.java

```diff
@@ -149,71 +148,49 @@ public void run() {
          }
          
-         String defaultOffsetDirection = NbBundle.getMessage(ViewDB.class,"CTL_DisplayOffsetDir");
-         String saved=Preferences.userNodeForPackage(TheApp.class).get("DisplayOffsetDir", defaultOffsetDirection);
-         Preferences.userNodeForPackage(TheApp.class).put("DisplayOffsetDir", saved);
-         
          String defaultGeometryPath = TheApp.getDefaultGeometrySearchPath();
-         saved=Preferences.userNodeForPackage(TheApp.class).get("Geometry Path", defaultGeometryPath);
+         String saved=TheApp.getCurrentVersionPreferences().get("Paths: Geometry Search Path", defaultGeometryPath);
          if (saved.isEmpty()||saved.equalsIgnoreCase("")){
              saved = defaultGeometryPath;
          }
          else if (!saved.contains(defaultGeometryPath) && updateResources)
              saved = saved.concat(File.pathSeparator+defaultGeometryPath);
-         Preferences.userNodeForPackage(TheApp.class).put("Geometry Path", saved);
+         TheApp.getCurrentVersionPreferences().put("Paths: Geometry Search Path", saved);
          // Push changes to API side
          GeometryFileLocator.updateGeometrySearchPathsFromPreferences();
-         String defaultBgColor = NbBundle.getMessage(OpenSimBaseCanvas.class, "CTL_BackgroundColorRGB");        
-         saved = Preferences.userNodeForPackage(TheApp.class).get("BackgroundColor", defaultBgColor);
-         Preferences.userNodeForPackage(TheApp.class).put("BackgroundColor", saved);
 
-         String muscleRadius = NbBundle.getMessage(ViewDB.class, "CTL_MuscleRadius");        
-         saved = Preferences.userNodeForPackage(TheApp.class).get("Muscle Display Radius", muscleRadius);
-         Preferences.userNodeForPackage(TheApp.class).put("Muscle Display Radius", saved);
+         String muscleRadius = "8";
+         saved = TheApp.getCurrentVersionPreferences().get("Visualizer: Muscle Display Radius (mm)", muscleRadius);
+         TheApp.getCurrentVersionPreferences().put("Visualizer: Muscle Display Radius (mm)", saved);
 
-         String experimentalMarkerDisplayScaleStr="1.0";
-         saved=Preferences.userNodeForPackage(TheApp.class).get("Experimental Marker Size", experimentalMarkerDisplayScaleStr);
-         Preferences.userNodeForPackage(TheApp.class).put("Experimental Marker Size", saved);
+         String experimentalMarkerDisplayScaleStr="5.0";
+         saved=TheApp.getCurrentVersionPreferences().get("Visualizer: Experimental Marker Radius (mm)", experimentalMarkerDisplayScaleStr);
+         TheApp.getCurrentVersionPreferences().put("Visualizer: Experimental Marker Radius (mm)", saved);
          
          String persistModels = "Off";        
-         saved = Preferences.userNodeForPackage(TheApp.class).get("Persist Models", persistModels);
-         Preferences.userNodeForPackage(TheApp.class).put("Persist Models", saved);
+         saved = TheApp.getCurrentVersionPreferences().get("Application: Persist Models", persistModels);
```

Comment:
Not sure what "persist models" means. Perhaps "Restore Models on Startup", or are you providing more meaningful descriptions in tooltips on hover?